### ISSUE TITLE
Unable to run PDF example in MacOS

### ISSUE THREAD

**Issue by wavded** [go-rod/rod#312]
**Rod Version:**  v0.85.8
**MacOS Version:** 11.1 (Intel)
**Go Version:** 1.15.5

## Steps to reproduce

Attempted to run this code based on [this example](https://github.com/go-rod/rod/blob/master/examples_test.go#L235-L254):

```go
package main

import (
	"github.com/go-rod/rod"
)

func main() {
	page := rod.New().MustConnect().MustPage("https://github.com").MustWaitLoad()

	// simple version
	page.MustPDF("my.pdf")
}
```

First, I received a panic standing permissions denied to launch the downloaded Chromium:

```
13:27 $ go run main.go
Download: https://storage.googleapis.com/chromium-browser-snapshots/Mac/818858/chrome-mac.zip
Progress: 00% 15% 36% 56% 77% 97% 100%
Downloaded: /var/folders/gg/5g93xzk53p74v0lxjzpx7trw0000gn/T/rod/chromium-818858.zip
Unzip to: /var/folders/gg/5g93xzk53p74v0lxjzpx7trw0000gn/T/rod/chromium-818858
Progress: 00% 36% 68% 100%

panic: fork/exec /var/folders/gg/5g93xzk53p74v0lxjzpx7trw0000gn/T/rod/chromium-818858/chrome-mac/Chromium.app/Contents/MacOS/Chromium: permission denied

goroutine 1 [running]:
github.com/go-rod/rod/lib/utils.E(...)
        /Users/wavded/Projects/go/pkg/mod/github.com/go-rod/rod@v0.85.8/lib/utils/utils.go:63
github.com/go-rod/rod/lib/launcher.(*Launcher).MustLaunch(0xc000110780, 0xc0000a2000, 0x162efa0)
        /Users/wavded/Projects/go/pkg/mod/github.com/go-rod/rod@v0.85.8/lib/launcher/launcher.go:305 +0xcd
github.com/go-rod/rod.(*Browser).Connect(0xc0001640f0, 0xa0, 0xc00010fd38)
        /Users/wavded/Projects/go/pkg/mod/github.com/go-rod/rod@v0.85.8/browser.go:144 +0x428
github.com/go-rod/rod.(*Browser).MustConnect(0xc0001640f0, 0xc00014aab0)
        /Users/wavded/Projects/go/pkg/mod/github.com/go-rod/rod@v0.85.8/must.go:27 +0x2f
main.main()
        /Users/wavded/Projects/dqm-api/rod/main.go:10 +0x2bb
exit status 2

```

Then, I manually installed that version of Chromium to get around the permissions issue and received this panic:

```
13:25 $ go run main.go
panic: invalid character '.' after top-level value

goroutine 1 [running]:
github.com/go-rod/rod/lib/utils.E(...)
        /Users/wavded/Projects/go/pkg/mod/github.com/go-rod/rod@v0.85.8/lib/utils/utils.go:63
github.com/go-rod/rod/lib/proto.RuntimeRemoteObjectID.ExecutionID(0xc000308560, 0x17, 0x143cbe0)
        /Users/wavded/Projects/go/pkg/mod/github.com/go-rod/rod@v0.85.8/lib/proto/patch.go:165 +0x119
github.com/go-rod/rod.(*Page).updateJSCtxID(0xc000122ab0, 0x1628da0, 0x1869c50)
        /Users/wavded/Projects/go/pkg/mod/github.com/go-rod/rod@v0.85.8/page_eval.go:327 +0x591
github.com/go-rod/rod.(*Page).initSession(0xc000122ab0, 0xc000010130, 0x10)
        /Users/wavded/Projects/go/pkg/mod/github.com/go-rod/rod@v0.85.8/page_eval.go:231 +0x18f
github.com/go-rod/rod.(*Browser).PageFromTarget(0xc0001b80f0, 0xc000016d60, 0x20, 0x0, 0x0, 0x0)
        /Users/wavded/Projects/go/pkg/mod/github.com/go-rod/rod@v0.85.8/browser.go:275 +0x2f0
github.com/go-rod/rod.(*Browser).Page(0xc0001b80f0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
        /Users/wavded/Projects/go/pkg/mod/github.com/go-rod/rod@v0.85.8/browser.go:193 +0x245
github.com/go-rod/rod.(*Browser).MustPage(0xc0001b80f0, 0x0, 0x0, 0x12b1617)
        /Users/wavded/Projects/go/pkg/mod/github.com/go-rod/rod@v0.85.8/must.go:45 +0xa9
main.main()
        /Users/wavded/Projects/dqm-api/rod/main.go:10 +0x2d1
exit status 2
```

Upon further digging it seems that the `MustPage` call is causing the panic.

## Expected behavior

First expected the downloaded Chromium to not encounter the permissions error.  Then, expected the example PDFs to be outputted.

**Comment by wavded** [comment]
`go.sum` output:

```
github.com/go-rod/rod v0.85.8 h1:Bl0gr856BaG/JOYcZBNikwwb7gpaXaTexwPERqzj6AI=
github.com/go-rod/rod v0.85.8/go.mod h1:UEYVhPXlrtQIL1Nlnq/fPY7Q5pB708UIEy5IKtNVuFg=
```

**Comment by ysmood** [comment]
I think it's a problem with the chrome version, can you share the info of the `chrome://version/` page?

**Comment by ysmood** [comment]
Should have been fixed in v0.85.9, if you have any problems please leave a comment.

**Comment by wavded** [comment]
This working now, thanks for the quick turnaround.
